### DISCUSSION TITLE
Debug frontend

### DISCUSSION THREAD

**Discussion by ramondepieri** [grafana/grafana#41125] [General]
Hi guys,

I am working with the source, because I want be a contributor to the project. I have already build the project and did some changes and everything is working well. But I am not able to debugger with Visual Studio Code. Some one that already did that can explain how can achieve that?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
